runtime/ftplugin/query.lua

```diff
@@ -2,5 +2,21 @@
 -- Language:	Tree-sitter query
 -- Last Change:	2022 Mar 29
 
+if vim.b.did_ftplugin == 1 then
+  return
+end
+
+local buf = vim.api.nvim_get_current_buf()
+if vim.g.query_lint_on and #vim.g.query_lint_on > 0 then
+  vim.api.nvim_create_autocmd(vim.g.query_lint_on or { 'BufWrite' }, {
```

Comment:
```suggestion
  vim.treesitter.start()
  vim.api.nvim_create_autocmd(vim.g.query_lint_on or { 'BufWrite' }, {
```

If we spin up a parser, might as well use it for highlighting (since the default legacy lisp syntax highlighting is pretty much useless here).